app/lib/elasticsearch/coreQuery.js

```diff
@@ -20,7 +20,6 @@ queryFilters[constants.searchTypes.kitsOver25] = {
   must: [
```

Comment:
the file has been renamed to coreQuery, but the exported name is still getFilter, should they be the same?

There's also a move away from having 'get' prefixes, i.e. `coreQuery` instead of `getCoreQuery`, and `baseQuery` instead of `getBaseQuery`.